### PR TITLE
we have lots of timeouts or long wait on these calls and it seems like w...

### PR DESCRIPTION
**Pull request by eishay**
...e can be more relaxed there

### CODE REVIEW

**Comment by eishay** [comment]
@stkem @leogrim

File: kifi-backend/modules/common/app/com/keepit/curator/CuratorServiceClient.scala [hunk 1]
```diff
@@ -119,7 +119,7 @@ class CuratorServiceClientImpl(
 
   def ingestPersonaRecos(userId: Id[User], personaIds: Seq[Id[Persona]], reverseIngestion: Boolean = false): Future[Unit] = {
     val payload = Json.obj("personaIds" -> personaIds)
-    call(Curator.internal.ingestPersonaRecos(userId, reverseIngestion), payload).map { _ => Unit }
+    call(Curator.internal.ingestPersonaRecos(userId, reverseIngestion), payload, callTimeouts = longTimeout).map { _ => Unit }
```
Comment (yingjieMiao) [inline kifi-backend/modules/common/app/com/keepit/curator/CuratorServiceClient.scala:122]: this is typically fast, i.e. under 1 second. Increase this means client side (e.g. mobile) `freezes` during sign up.  I see the timeouts are more frequent after we 'scaled out' curator. Likely to be a contention problem.

Comment (eishay) [inline kifi-backend/modules/common/app/com/keepit/curator/CuratorServiceClient.scala:122]: so you thing we should revert this one?

Comment (yingjieMiao) [inline kifi-backend/modules/common/app/com/keepit/curator/CuratorServiceClient.scala:122]: I think this is better for user experience. Let user waiting for 30 seconds is not fun. 

We used to have an async fashion, i.e. shoebox tells curator to inject and forget about it (without curator confirm it's done). The problem was client will immediately finish the sign up process and call curator for recos. Sometimes ingestion hasn't even started by then, which leaves a blank page.

**Comment by eishay** [comment]
ok, will revert that one

**Pull request merged**